features/support/helpers.rb

```diff
@@ -228,3 +228,11 @@ def get_table_rows_by_caption(caption)
   result_table_rows_xpath = result_table_xpath + rows_xpath
   page.all(:xpath, result_table_rows_xpath)
 end
+
+def get_summary_list_rows_by_preceding_heading(heading)
+  # do we even h5?
```

Comment:
I have grepped our frontend apps and cannot see any `h5`s.

Reply:
:tada: